apps/web/src/app/[locale]/(dashboard)/[wsId]/inventory/products/edit/form.tsx

```
@@ -0,0 +1,461 @@
+                      <FormMessage />
+                    </FormItem>
+                  )}
+                />
+              </div>
+            </CardContent>
+          </Card>
+
+          <Button type="submit" className="w-full" disabled={loading}>
+            {loading
+              ? t('common.processing')
+              : !!data?.id
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)